framework/console/controllers/BaseMigrateController.php

```diff
@@ -41,6 +41,28 @@
      * or a file path.
      */
     public $templateFile;
+    /**
+     * @var array the template file for generating migration code automatically.
+     * This can be either a path alias (e.g. "@app/migrations/template.php")
+     * or a file path.
+     */
+    public $templateFileGenerators;
+    /**
+     * @var array the fields
+     */
+    public $fields;
+    /**
+     * @var string the primary key name
+     */
+    public $primaryKey = 'id';
+    /**
+     * @var string
+     */
+    public $createAt = 'create_at';
```

Comment:
Yes. I think it should be dropped as well. For junction tables it's common to have PK for both colums:

```
post_tag
======
post_id
tag_id

PK (post_id, tag_id)
```